Sync/SyncTelemetryUtils.swift

```diff
@@ -384,3 +384,72 @@ public struct SyncPing: SyncTelemetryPing {
         }
     }
 }
+
+public class GleanSyncOperationHelper {
+    public init () {}
+
+    public func start() {
+        GleanMetrics.Sync.syncUuid.generateAndSet()
+    }
+
+    public func end(_ result: SyncOperationResult) {
+        if let engineResults = result.engineResults.successValue {
+            engineResults.forEach { result in
+                let (name, status) = result
+                switch status {
+                case .completed(let stats):
+                    self.submitPingPerEngine(name, stats)
+                case .partial(let stats):
+                    self.submitPingPerEngine(name, stats)
+                case .notStarted(let reason):
+                    ()
```

Comment:
nit: More standard on how to skip a switch case
```suggestion
                    break
```